### PR TITLE
fix(feature-card): make entire card clickable on mobile viewports

### LINKED ISSUE #7806: [cta | feature]: Card is not fully-clickable in mobile
###  Engineering info:
- HC engineer: @andy-blum 
- HC Jira Ticket: https://jsw.ibm.com/browse/HC-2347
- DDS consulting engineer: N/A

----

### Description

The feature variant of the CTA component is not fully-clickable across all browser sizes. In the web components package, the vertical layout (<672px wide) does not extend the clickable area over the image. In the react package, below the 400px breakpoint the image isn't part of the interactive space.

Screen Shot 2021-12-02 at 2 13 40 PM

Screen Shot 2021-12-02 at 2 13 51 PM

screen_shot_2021-12-02_at_2 20 39_pm

### Component(s) impacted

[React](https://ibmdotcom-react-canary.mybluemix.net/?path=/story/components-cta--feature)
[Web Components](https://ibmdotcom-web-components-canary.mybluemix.net/?path=/story/components-cta--feature)

### Browser

Chrome

### Carbon for IBM.com version

latest

### Severity

Severity 3 = The problem is visible or noticeable to users but does not impede the usability or functionality. Affects minor functionality, has a workaround.

### Application/website

carbon-for-ibmdotcom

### Package

@carbon/ibmdotcom-react, @carbon/ibmdotcom-web-components

### CodeSandbox example

see canary links above

### Steps to reproduce the issue (if applicable)

_No response_

### Release date (if applicable)

_No response_

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/master/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

### PR DESCRIPTION
**Pull request by andy-blum**
### Related Ticket(s)

Fixes #7806

### Description

This PR removes unnecessary declarations of `position:relative;` causing the link's `:after` pseudo-element (indicated by red region in below images) to only cover the card's content area but not the image area.

Before:
Screen Shot 2021-12-10 at 10 53 40 AM

After:
Screen Shot 2021-12-10 at 10 53 15 AM

### Changelog

**Changed**

- Makes entirety of feature-card component clickable on mobile

### CODE REVIEW

**Review by proeung — CHANGES_REQUESTED** [review]
I'm seeing a regression in the mobile breakpoints where the CTA hover stylings are not coming through (see screenshot). Hover should be consistent on all breakpoints.


https://user-images.githubusercontent.com/1815714/145628030-29203ed0-f275-489b-8058-8134a5b32b22.mov

@shixiedesign or @RichKummer Can you help take a look as well? Thanks!

**Comment by andy-blum** [comment]
@proeung What hover state are you comparing this to? The deploy preview and canary match up perfectly for me (brower tabs change at 0:06 below)

https://user-images.githubusercontent.com/25532785/145636930-9a433f3f-5352-4307-8574-ab25275a5c64.mov

**Comment by proeung** [comment]
@andy-blum Looks like the mobile hover state regression that I noted above is happening on the [WC canary preview](https://ibmdotcom-web-components-canary.mybluemix.net/?path=/story/components-cta--feature) as well, which means that it's not a regression coming from this particular PR. If we can, let's figure out what's causing this issue and push up a solution in this PR.

Card hover state should be `var(--cds-inverse-hover-ui, #4c4c4c)`.

cc: @IgnacioBecerra @oliviaflory @RichKummer

**Review by emyarod — APPROVED** [review]
looks good to me!

File: packages/styles/scss/components/feature-card/_feature-card.scss [hunk 1]
```diff
@@ -80,7 +80,7 @@ $fcb-breakpoint-up--lg: map-get(
         }
       }
 
-      .#{$prefix}--card__content {
+      .bx--card__wrapper {
```
Comment (proeung) [inline packages/styles/scss/components/feature-card/_feature-card.scss:83]: Minor nitpicking, but let's use the `$prefix` variable.

```suggestion
      .#{$prefix}--card__wrapper {
```

**Comment by proeung** [comment]
Looks good overall! Thanks for fixing the hover state issue seen the mobile breakpoints. I just have one minor feedback and I think this PR should be good for merging unless we need further QA from the design team.

**Review by proeung — APPROVED** [review]

**Comment by annawen1** [comment]
@andy-blum Percy caught this happening in the `Back to Top` story where the feature card is being used in 320px screen size where the icon is going out of the card. Is that a quick fix?

Screen Shot 2021-12-22 at 10 19 13 AM

**Comment by andy-blum** [comment]
While we could refactor the card styles to make this adapt to fit that text, content guidance for this component directs a maximum of 65 characters and this implementation has 90 characters.

**Review by annawen1 — APPROVED** [review]
LGTM! Thanks @andy-blum!

**Pull request merged**